content/user/ezpages/ezpages_display.md

```diff
@@ -14,45 +14,39 @@ EZ-Page links can be shown without making a code change in three places:
 - Important Links sidebox
 - Footer 
 
-A set of configurations determines where these links are shown.  If you go to
-Admin > Tools > EZ-Pages, you will see a set of fields like the following: 
+A set of configurations determines where these links are shown.  There is a radio button and sort order for each of the header,
+sidebox and footer. 
+You can see these settings in [Tools > EZ-Pages](/user/admin_pages/tools/ezpages/). 
 
-<br />
-<img src="/images/ezpages.png" alt="EZ-Pages configuration" />
-<br />
+# Adding a link manually 
 
-- Setting Header = Yes with a sort order > 0 means a link to this page will be displayed by `includes/templates/responsive_classic/templates/tpl_ezpages_bar_header.php`. 
-
-- Setting Sidebox = Yes with a sort order > 0 means a link to this page will be displayed by `includes/modules/sideboxes/ezpages.php`. 
-
-- Setting Footer = Yes with a sort order > 0 means a link to this page will be displayed by `includes/templates/responsive_classic/templates/tpl_ezpages_bar_footer.php`. 
-
-If you do not want the link to appear in any of these places or you are not using these components in your template, you still have the ability to  show an EZ-Page link and have the EZ-Page display properly: 
-
-- use the `Page is Visible` radio button 
-- code into the content where you want the link to appear 
+In addition to the three places shown above, you can change your
+template to show an EZ-Page using the technique below. 
+Let's suppose the language constant for the 
+name of the page is `NEW_PAGE_TITLE` and the EZ-Page id, as shown on 
+the left hand side of 
+[Tools > EZ-Pages](/user/admin_pages/tools/ezpages/), is 19. 
 
 If you are already in a PHP block, you'd use: 
 
 ```
-  echo '<a href="' . zen_href_link(FILENAME_EZPAGES,'page_id') . '">' NEW_PAGE_TITLE . '</a>'; 
+  echo '<a href="' . zen_href_link(FILENAME_EZPAGES,'id=19') . '">' NEW_PAGE_TITLE . '</a>'; 
```

Comment:
Missing a `.` before NEW_PAGE_TITLE

Reply:
Ugh - second coding error on that page. :(. Fixed now.